### PR TITLE
recipes: Update LICENSE variable to use SPDX license identifiers

### PR DESCRIPTION
**Pull request by matthiasklein**
Signed-off-by: Matthias Klein <matthias@extraklein.de>

### CODE REVIEW

File: recipes-bsp/dt-utils/dt-utils.inc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 DESCRIPTION = "device-tree and barebox-related tools"
 HOMEPAGE = "http://git.pengutronix.de/?p=tools/dt-utils.git"
 SECTION = "base"
-LICENSE = "GPLv2"
+LICENSE = "GPL-2.0-only"
 LIC_FILES_CHKSUM = "file://COPYING;md5=9ac2e7cff1ddaf48b6eab6028f23ef88"
```
Comment (rohieb) [inline recipes-bsp/dt-utils/dt-utils.inc:5]: I would also include the snippet from README here as source for the `-only` part:
`file://README;beginline=39;md5=f139f662160ab6bc7eb412251d89d885`

File: recipes-casync/casync/casync_git.bb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 SUMMARY = "Content-Addressable Data Synchronization Tool"
 HOMEPAGE = "https://github.com/systemd/casync"
-LICENSE = "LGPLv2.1"
+LICENSE = "LGPL-2.1-only"
```
Comment (rohieb) [inline recipes-casync/casync/casync_git.bb:3]: Hmm, at least https://github.com/systemd/casync/blob/master/src/cacache.c says "LGPL2.1+", that would be `-or-later`…?

File: recipes-core/rauc/rauc.inc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 DESCRIPTION = "RAUC update controller for host and target"
 HOMEPAGE = "https://github.com/rauc/rauc"
-LICENSE = "LGPLv2.1"
+LICENSE = "LGPL-2.1-only"
```
Comment (rohieb) [inline recipes-core/rauc/rauc.inc:3]: Hmmm. I could only a LGPL license text, but no sign of `-or-later` or `-only` in any of the RAUC code, … @jluebbe, @ejoerns, could you clarify?

File: recipes-support/rauc-hawkbit/rauc-hawkbit_git.bb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-LICENSE = "LGPLv2.1"
+LICENSE = "LGPL-2.1-only"
```
Comment (rohieb) [inline recipes-support/rauc-hawkbit/rauc-hawkbit_git.bb:1]: The README says `either version 2.1 of the License, or (at your option) any later version`, I would add it to LIC_FILES_CHKSUM too: `file://README.rst;beginline=114;endline=132;md5=a569a9868bb81fb591cf2cdfdf61db1d`

**Review by rohieb — CHANGES_REQUESTED** [review]

File: recipes-core/rauc/rauc.inc (hunk 3 above)
Comment (jluebbe) [reply recipes-core/rauc/rauc.inc:3]: We don't have any `-only` restriction.

**Comment by matthiasklein** [comment]
@rohieb I have reworked the PR according to your comments.

**Comment by matthiasklein** [comment]
@rohieb The README file of dt-utils version 2021.03.0 does not have a relevant license section yet: https://git.pengutronix.de/cgit/tools/dt-utils/tree/README?h=v2021.03.0

**Comment by rohieb** [comment]
> @rohieb The README file of dt-utils version 2021.03.0 does not have a relevant license section yet: https://git.pengutronix.de/cgit/tools/dt-utils/tree/README?h=v2021.03.0

Ah. Yes, my bad. I looked on the current master :-)

**Review by ejoerns — APPROVED** [review]

**Review by rohieb — APPROVED** [review]

**Pull request merged**